Makefile.am

```diff
@@ -137,6 +137,15 @@ EXTRA_DIST += \
 CLEANFILES += \
     $(man1_MANS)
 
+### initramfs hooks ###
+
+if HAVE_DRACUT
+if PLYMOUTH
+dracut_DATA = dist/dracut/module-setup.sh dist/dracut/tpm2-totp.sh
+endif # PLYMOUTH
+endif # HAVE_DRACUT
+EXTRA_DIST += dist/dracut/tpm2-totp.sh
```

Comment:
Shouldn't this be in the tarball already due to appearing in https://github.com/tpm2-software/tpm2-totp/blob/b2d346ec895f46cefca7ac1891ff83d9718ee439/Makefile.am#L144

Reply:
Data [isn't included in the distribution by default](https://www.gnu.org/software/automake/manual/html_node/Data.html), that's why the `EXTRA_DIST` is necessary. We could automatically add it to the distribution by using `dist_dracut_DATA`, but then `dist/dracut/module-setup.sh` would be distributed as well, which is not what we want: it is generated from `dist/dracut/module-setup.sh.in` (which is in `AC_CONFIG_FILES` and therefore distributed automatically), so we don't need the auto-generated artefact.

Reply:
Ok, nevermind then